src/Aurora.Application/Extensions/LoggerExtensions.cs

```diff
@@ -11,9 +11,9 @@ public static class LoggerExtensions
         public static void LogRequest(this ILogger logger, SearchRequestDto request, string prefix)
         {
             logger.LogInformation(
-            "{prefix} for request for '{term}' term with '{types}' types in '{websites}' websites",
+            "{prefix} for request for '{term}' terms with '{types}' types in '{websites}' websites",
             prefix,
-            request.SearchTerm,
+            request.SearchTerms.CommaSeparate(),
```

Comment:
why dont make it generic separation but select separation charactre d???  ??  ? ?

Reply:
For easier end-user usage